tests/nopytest_test_coadds.py

```diff
@@ -80,72 +80,24 @@
 DATAREPO_ROOT = os.path.join(os.path.dirname(__file__), ".tests", "testCoadds-data")
 
 
+def assertWrapper(func):
+    """Decorator to intercept AssertionError and reraise whilst recording
+    a failure."""
+    def wrapped(self):
+        try:
+            func(self)
+        except Exception:
```

Comment:
Should we be catching `Exception` here, or `AssertionError` (per the docstring, above)?

Reply:
I did have AssertionError here in the lsst_ci variant but then I got to pondering that any failures whilst running a test would also be a good reason not to delete. I can fix the docstring.